kornia/geometry/conversions.py

```diff
@@ -1175,3 +1185,163 @@ def normalize_homography3d(
     # compute chain transformations
     dst_norm_trans_src_norm: torch.Tensor = dst_norm_trans_dst_pix @ (dst_pix_trans_src_pix @ src_pix_trans_src_norm)
     return dst_norm_trans_src_norm
+
+
+def extrinsics_from_Rt(R: Tensor, t: Tensor) -> Tensor:
+    r"""Combines 3x3 R and 1x3 t into 4x4 extrinsics.
+    Extrinsics matrix is usually named `K`, but we avoid this,
+    to not interfere with `import kornia as K`
+
+    Args:
+        R: Rotation matrix, :math:`(B, 3, 3).`
+        t: Translation matrix :math:`(B, 3, 1)`.
+
+    Returns:
+        the extrinsics :math:`(B, 4, 4)`.
+
+    """
+    KORNIA_CHECK_SHAPE(R, ["B", "3", "3"])
+    KORNIA_CHECK_SHAPE(t, ["B", "3", "1"])
+    Rt = torch.cat([R, t], dim=2)
```

Comment:
```suggestion
    Rt = concatenate([R, t], -1)
```
`from konia.core import concatenate`

towards numpy like api

Reply:
I actually hate this particular part of numpy :)